src/Common/perf/PerfRunner/PerfRunner.cs

```diff
@@ -0,0 +1,30 @@
+using System.IO;
+using System.Reflection;
+using System.Collections.Generic;
+using Microsoft.Xunit.Performance.Api;
+
+public class PerfHarness
+{
+    public static void Main(string[] args)
+    {
+        using (XunitPerformanceHarness harness = new XunitPerformanceHarness(args))
```

Comment:
Can you confirm that this doesn't create a gigantic ETL file that will cause us to have to break these up?  I believe the previous implementation that ran in Helix collected one ETL per test assembly.